src/m365/planner/commands/task/task-list.ts

```diff
@@ -161,6 +180,17 @@ class PlannerTaskListCommand extends GraphItemsListCommand<Task> {
       });
   }
 
+  private mergeTaskPriority(taskItems: any[], betaTaskItems: any[]): any[] {
```

Comment:
Let's avoid using `any` to avoid runtime errors. Instead we can use the Task type